### ISSUE TITLE
Question about the loss function!

### ISSUE THREAD

**Issue by ZonghanZhang** [qqwweee/keras-yolo3#139]
Hi, I have just read your codes and found that you used the raw data of bounding boxes and the sigmoid _cross_entropy the loss for the x,y,w,h, confidence and classes. 
I am a little bit confused:
1. Is there any advantages of using the raw labels instead of the transformed labels to calculate the loss?
2.The author used square loss for all labels, so did you find out that the entropy loss is better than the square loss?

I really benefit a lot from your codes, thank you in advance！
